examples/ifc/ifc.ttl

```diff
@@ -1,24 +1,24 @@
 @prefix bldg: <urn:example#> .
 @prefix brick: <https://brickschema.org/schema/Brick#> .
-@prefix brickifc: <https://brickschema.org/extension/ifc#> .
+@prefix ref: <https://brickschema.org/schema/Brick/ref#> .
 
-bldg:ifc_project a brickifc:Project ;
-    brickifc:projectID "abc" ;
-    brickifc:fileLocation "file://./building.ifc" ;
+bldg:ifc_project a ref:ifcProject ;
+    ref:ifcProjectID "abc" ;
+    ref:fileLocation "file://./building.ifc" ;
 .
 
 bldg:space a brick:Space ;
-    brick:hasIFCReference [
-        brickifc:hasProjectReference bldg:ifc_project ;
-        brickifc:globalID "123" ;
-        brickifc:name "Example Space" ;
+    ref:hasExternalReference [
+        ref:hasIfcProjectReference bldg:ifc_project ;
+        ref:ifcGlobalID "123" ;
+        ref:ifcName "Example Space" ;
     ] .
 
 bldg:space2 a brick:Space ;
-    brick:hasIFCReference [
-        brickifc:hasProjectReference bldg:ifc_project ;
-        brickifc:globalID "124" ;
-        brickifc:name "Example Space" ;
+    ref:hasExternalReference [
```

Comment:
I understand the type of this reference can be inferred, but still it'd be useful to have the type for users.

Reply:
Users can add the type to the object of `ref:hasExternalReference` but it can also be inferred

Reply:
Yeah, as I said, I understand the type can be inferred. I think it's useful to have the type explicitly in this specific example because it's easier for users to read. For those not familiar with the inference logic which is embedded deep in the ttl file, it's just easier to see the type of the node explicitly. Without it, one may think the properties can be arbitrarily added here.